### PR TITLE
docs: cleanup

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [x] Documentation content changes
 - [ ] Other, please describe:

## Impact on version

- [ ] **Major**
  - The template structure was changed.
- [ ] **Minor**
  - New functionalities were added.
- [x] **Patch**
  - A bug in behavior was fixed.
  - Documentation was changed.

### Always applicable
No matter your changes, these checks always apply.
- [x] Your conventional commits are aligned with the **Impact on version** section.
- [x] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [x] Documentation files were updated according with the changes.
  - Update `README.md` and `src/cli/CLI.md` if you made changes to **templating**.
  - Update `AzurePipelines.md` and `src/app/README.md` if you made changes to **pipelines**.
  - Update `Diagnostics.md` if you made changes to **diagnostic tools**.
  - Update `Architecture.md` and its diagrams if you made **architecture decisions** or if you introduced new **recipes**.
  - ...and so forth: Make sure you update the documentation files associated to the recipes you changed. Review the topics by looking at the content of the `doc/` folder.
- [x] Images about the template are referenced from the [wiki](https://github.com/nventive/FlutterApplicationTemplate/wiki/Images) and added as images in this git.
- [x] TODO comments are hints for next steps for users of the template and not planned work.

### Contextual
Based on your changes these checks may not apply.
- [x] Automated tests for the changes have been added/updated.
- [x] Tested on all relevant platforms

### CODE REVIEW

**Review by Marc-Antoine-Soucy — APPROVED** [review]

File: doc/AzurePipelines.md [hunk 1]
```diff
@@ -22,16 +22,22 @@ At high level, the CI/CD pipelines do the following:
 It also runs automated tests during the build steps.
 
 ### Pull request runs
-Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**. To reduce the build time, some stages and steps are disabled for PR builds. This requires a specific variable called `IsLightBuild` to be set, hence why it is appearing in the pipeline. 
+
+Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**.
+To reduce the build time, some stages and steps are disabled for PR builds.
```
Comment (jeanplevesque) [inline doc/AzurePipelines.md:27]: Is this true?

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: Yes, iOS.

Comment (jeanplevesque) [reply doc/AzurePipelines.md:27]: Why do you mean by just "iOS"?

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: The iOS stage

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: and the release stages

Comment (jeanplevesque) [reply doc/AzurePipelines.md:27]: Well obviously we don't release anything when doing PR validation. But are there other build settings that change?

Comment (jeanplevesque) [reply doc/AzurePipelines.md:27]: And do you have the `IsLightBuild` variable?

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: Yes

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: > Well obviously we don't release anything when doing PR validation. But are there other build settings that change?

Nah it's light build or not and based on environment if I remember correctly

Comment (jeanplevesque) [reply doc/AzurePipelines.md:27]: In .NET projects, `IsLightBuild` is used to change the build configuration (such as disabling AOT) to speed it up. If the only thing that is different is whether or not to run the deployment stages, I think the `IsLightBuild` variable doesn't make sense.

Comment (Soap-141) [reply doc/AzurePipelines.md:27]: May be the name doesn't but we can still use it, any CI refactoring should be done in another PR right? I just cleaned up the docs.

Comment (jeanplevesque) [reply doc/AzurePipelines.md:27]: sure

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**